test/fixtures/rules/std_rule_cases/L009.yml

```diff
@@ -0,0 +1,12 @@
+rule: L009
```

Comment:
Wow, I'm surprised this rule didn't have any `.yml` tests before now!

Reply:
Better late than never!